src/promptflow/promptflow/core/_flow.py

```diff
@@ -259,7 +278,7 @@ def __init__(
         # entry file name
         self.entry_file = self._resolve_entry_file(entry=entry, working_dir=code)
         # TODO(2910062): support eager flow execution cache
-        super().__init__(data=data, path=path, code=code, content_hash=None, **kwargs)
+        super().__init__(code=code, path=path, dag=data, content_hash=None, **kwargs)
```

Comment:
why this change？
why it has dag parameter here?

Reply:
We have change this FlexFlow base class against Flow.

So the super() now is no longer FlowBase but Flow.

Flow init contains the paramter dag.
image.png

Reply:
![image](https://github.com/microsoft/promptflow/assets/2208599/6f89a1c5-57a2-439f-bbaa-0160118f6429)